src/widgets/panel/panel.tsx

```diff
@@ -96,27 +123,25 @@ const renderer = ({
   );
 };
 
-type NestedWidgetOptions = {
-  container: HTMLElement | string;
-  [key: string]: any;
-};
-type PanelWidget = (
-  params?: PanelWidgetParams
-) => (
-  widgetFactory: WidgetFactory<{}, NestedWidgetOptions>
-) => (widgetOptions: NestedWidgetOptions) => Widget;
+export type PanelWidget = (
+  params: PanelWidgetOptions
```

Comment:
I guess `PanelWidgetOptions` should be renamed to `PanelWidgetParams`?

Reply:
I am not sure, because for types is always have been WidgetOptions.

Anyways, going to address of all of that together.

Just created this task for that: https://algolia.atlassian.net/browse/IFW-1260?atlOrigin=eyJpIjoiZjQ3ODg0ZDg0YmVmNDEzNmJjZTA1ZTAzYWRhNDEyMTEiLCJwIjoiaiJ9